dali/operators/math/expressions/arithmetic.h

```diff
@@ -155,7 +155,7 @@ inline std::vector<ExprImplTask> CreateExecutionTasks(const ExprNode &expr, Expr
   return result;
 }
 
-inline TensorListShape<> ShapePromotion(std::string op, span<const TensorListShape<> *> shapes,
+inline TensorListShape<> ShapePromotion(const std::string &op, span<const TensorListShape<> *> shapes,
```

Comment:
```suggestion
inline TensorListShape<> ShapePromotion(const std::string &op,
                                        span<const TensorListShape<> *> shapes,
```
The line length limit of 100 was exceeded.